artifactory/utils/dependenciesutils.go

```diff
@@ -46,7 +46,7 @@ func DownloadExtractorIfNeeded(downloadPath, targetPath string) error {
 	if exists || err != nil {
 		return err
 	}
-
+	log.Debug("build-info-extractor does not  exist locally. Downloading the relevant jar")
```

Comment:
Let's change this message from Debug to Info.
Also, let's rephrase from:
build-info-extractor does not  exist locally. Downloading the relevant jar
to:
The build-info-extractor jar is not cached locally. Downloading it now... You can set the repository from which this jar is downloaded. Read more about it at https://www.jfrog.com/confluence/display/CLI/CLI+for+JFrog+Artifactory#CLIforJFrogArtifactory-DownloadingtheMavenandGradleExtractorJARs